ecosystem/sep-0006.md

```diff
@@ -943,7 +1000,7 @@ All assets listed in a `deposit` and `deposit-exchange` can contain these attrib
 
 * `enabled`: `true` if SEP-6 deposit for this asset is supported
 * `authentication_required`: Optional. `true` if client must be [authenticated](#authentication) before accessing the deposit endpoint for this asset. `false` if not specified.
-* `fields` object as explained below.
+* `fields` (**Deprecated**, KYC information should be supplied to the Anchor via SEP-12) `fields` object is explained below.
```

Comment:
We might want to expand on why this is deprecated (security of passing PII in URLs).